### ISSUE TITLE
Gray screen

### ISSUE THREAD

**Issue by acondr2** [chasemc/IDBacApp#38]
About 3 weeks ago I was working with IDBac and was having difficulties proceeding through the workflow. The program seemed to "crash" before starting m/z conversion. While selecting a directory, raw data, or excel sheet, the screen would turn gray and I would be unable to click convert or proceed in any way in the program.  I would have to close and restart the process multiple times until it would finally work.

in some instances, the same issue persisted even after conversion began. Sometimes the program would stop mid processing or conversation and I would have to close and restart again.

Out of about 15 attempts, three analysis completed fully and I was able to view and download my data.

**Comment by chasemc** [comment]
Thanks and sorry for the troubles, it sounds frustrating.

 Unfortunately that's going to be hard to diagnose remotely, even whether it's an issue with IDBac, R, or some setting of the computer itself.  

If this happens again, would you mind emailing me the log (right-click the desktop shortcut, select "Open file location" and it's the file within the "log" folder.

**Comment by acondr2** [comment]
Okay! I will be trying to run some stuff again today/tomorrow so I will do
that 👍

Best,
Alanna Condren
*EYH Chicago Co-Director*
*PhD Candidate*
College of Pharmacy
University of Illinois at Chicago
Sanchez Lab Website <https://www.sanchezlab.science/>

**Comment by acondr2** [comment]
Chase!

Just thought of something that might be nice to add to IDBac.

After Sofia did 16S for me I swapped out some of the nomenclature I have
for my library strains in my excel spreadsheet for the genus and species
Sofia confirmed with 16S. That way the dedogram would mix together the
strains we have characterized and the uncharacterized ones. However, after
running I noticed only the genus was used int he dedorgam. For example, I
had multiple bacillus species (B.subtilis & B. coagulans) in my IDBac run
but it combined all samples that started with Bacillus in my excel
spreadsheet into one branch instead of putting them as neighbors (as they
should be) but separated by species.

Maybe this is something I can just fix by adding a underscore between the
genus and species but I would prefer to not have that in the dedogram since
I would like to use it as a figure in a paper. I can always go back and
edit in inkscape I guess but I thought I would let you know this is
something minor I came across.

If my explanation is confusing let me know and maybe I can run over to MBRB
and explain it better!

Best,
Alanna Condren
*EYH Chicago Co-Director*
*PhD Candidate*
College of Pharmacy
University of Illinois at Chicago
Sanchez Lab Website <https://www.sanchezlab.science/>
